service/arm_templates/private_load_balancer_setup.json

```diff
@@ -1,114 +0,0 @@
-{
```

Comment:
This template is not required anymore since we don't create any private load balancer.